docker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine
+
+RUN apk add --no-cache libgcc libxml2 readline bash
+
+RUN set -xe ; \
+    apk add --no-cache --virtual=.build gcc libc-dev make bison flex readline-dev libxml2-dev git automake autoconf libtool pkgconf coreutils \
+ && git clone git://github.com/hercules-team/augeas \
```

Comment:
Why clone it instead of copying from the local dir?

Reply:
I was asking myself the same question. Is it better to build the image with the dockerfile alone or with the full project?

I can change it to copy from the local dir.